.github/workflows/test_code.yaml

```diff
@@ -20,7 +28,22 @@ jobs:
       - name: Install hatch
         run: pip install hatch
       - name: Test Python
-        run: hatch run test:test
+        run: hatch run test:cov
+
+      - name: Coverage comment
+        id: coverage_comment
+        uses: py-cov-action/python-coverage-comment-action@v3
+        with:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Store Pull Request comment to be posted
```

Comment:
Does this post the comment? If so maybe the name could be `Add coverage summary to pull request` or similar?